src/SourceBuild/content/prep.sh

```diff
@@ -53,6 +54,9 @@ while :; do
     --no-sdk)
       installDotnet=false
       ;;
+    --artifacts)
```

Comment:
This argument should be documented.

I'm not a fan of the name as it doesn't feel self-descriptive to me.  Thoughts on using `artifacts-rid` instead?

Reply:
Renamed to [artifacts-rid](https://github.com/dotnet/installer/pull/17370/files#diff-a1d21cbd7f180cea469d4fb634d555679ca532f21d903b2b5df311f6a051aa6aR59) for the script and [artifactsRid](https://github.com/dotnet/installer/pull/17370/files#diff-dc549eeb99aa73960e74c9d4c160a3c76e3e414398bb348c1d5365aac0369ad5R24) for the pipeline parameter